omisego-sdk/src/main/java/co/omisego/omisego/constant/Headers.kt

```diff
@@ -0,0 +1,15 @@
+package co.omisego.omisego.constant
+
+/**
+ * OmiseGO
+ *
+ * Created by Phuchit Sirimongkolsathien on 5/3/2018 AD.
+ * Copyright © 2017-2018 OmiseGO. All rights reserved.
+ */
+
+object Headers {
```

Comment:
Maybe it would be worth to precise `HttpHeaders` (or `HTTPHeaders`) ? Just raising the question :)

Reply:
Fixed